File: Codechef/CHEFWARS.py
#----------------------CODE-------------------#
#-----------------------BY-------------------#
#-------------------ABHISHEK RAJ-------------------#
for i in range(int(input())):
    d,c=[int(x) for x in input().split()]
#     print(d,c)
    while (d>0 and c>0):
        d=d-c
#         print('d',d)
        c=(c//2)
#         print('c',c)
    if c==0 and d<=0:
        print("1")
    elif c==0:
        print("0")
    elif d<=0:
        print("1")

        

File: Codechef/CHEFWARS.cpp
#----------------------CODE-------------------#
#-----------------------BY-------------------#
#-------------------ABHISHEK RAJ-------------------#
#include <iostream>
using namespace std;

int main() {
	int t;
	cin>>t;
    while(t--){
        int d;
        int c;
        cin>>d>>c;
        while(d>0 && c>0){
            d=d-c;
            c=c//2;
        }
        if c==0 && d<=0{
            cout<<1<<endl;
        }
        else if (c==0){
            cout<<0<<endl;
        }
        else if d<=0{
            cout<<1<<endl;
        }
    }
